src/utils/search.ts

```diff
@@ -396,6 +416,113 @@ export class Search {
    * `skipT*` filters and the `schemaFilters`.
    */
 
+  debugRecentItems() {
+    const recents = this._loadAndCleanRecentItems();
+    return recents;
+  }
+
+  private _loadAndCleanRecentItems(): StoredRecentItem[] {
+    try {
+      const raw = localStorage.getItem(this.recentKey);
+      if (!raw) {
+        return [];
+      }
+
+      const parsed: StoredRecentItem[] = JSON.parse(
+        raw ?? '[]'
+      ) as StoredRecentItem[];
+
+      return parsed;
```

Comment:
```suggestion

      return JSON.parse(
        raw ?? '[]'
      ) as StoredRecentItem[];
```

You can directly return it